docs/installation.rst

```diff
@@ -5,14 +5,16 @@
 Installation
 ============
 
-To install |project_name|, you need `Python 3`_ and `pip`_ installed. You
-can then either
-`download it directly
+To install Borg-Import, you need `Python 3` and `pip` installed. You
+can then either `download borg-import directly
 <https://github.com/borgbackup/borg-import/archive/master.zip>`_
-or clone it from github_.
-Extract the content, open a terminal in its root folder and
-execute the following to install it via pip:
-:code:`pip install --user -e .`
+or clone it from `github<https://github.com/borgbackup/borg-import>`_.
+Unzip the downloaded .zip if necessary.
```

Comment:
.tar.gz is also something that is frequently used, so I'ld prefer keeping the old phrase "extract the content".